### PR TITLE
code optimisation

### PR DESCRIPTION
**Pull request by owner6**

### CODE REVIEW

File: data-analysis.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+function getData(url) {
+	return fetch(url)
+		.then(response => response.json())
+		.then(data => {
+		return data.users;
+		})
+	}
+
+function processData(users) {
```
Comment (dever4eg) [inline data-analysis.js:17]: You can split this function into multiple function like getMinWeightUser, getAverageHeight, ...

File: data-analysis.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+	.then(users => {
+		const results = processData(users);
+		displayResults(results);
+	})
+
+function getData(url) {
+	return fetch(url)
+		.then(response => response.json())
+		.then(data => {
+		return data.users;
```
Comment (dever4eg) [inline data-analysis.js]: One tab is missing here

File: data-analysis.js [hunk 3]
```diff
@@ -0,0 +1,76 @@
+const url = 'https://dummyjson.com/users';
+getData(url)
```
Comment (dever4eg) [inline data-analysis.js:3]: I would like to have one empty line after const

**Review by dever4eg — CHANGES_REQUESTED** [review]

File: data-analysis.js (hunk 1 above)
Comment (owner6) [reply data-analysis.js:17]: I can try to do this in the next branch when I make the next pull request

File: data-analysis.js (hunk 2 above)
Comment (owner6) [reply data-analysis.js]: corrected

File: data-analysis.js (hunk 3 above)
Comment (owner6) [reply data-analysis.js:3]: an empty string is added

**Review by dever4eg — APPROVED** [review]

File: data-analysis.js (hunk 1 above)
Comment (dever4eg) [reply data-analysis.js:17]: Okay, approved

**Pull request merged**